### DISCUSSION TITLE
Add "Emission angle" in the particle object editor

### DISCUSSION THREAD

**Discussion by Bouh** [4ian/GDevelop#3639] [Enhancement ideas (and their technical discussions)]
## Description

There is no way to configure the **Emission angle** in object properties of a particle object.
The only way is to use the action.

## Solution suggested

Add an input to configure "Emission angle" in the particle object.

**Comment by tristanbob** [comment]
The angle of a particle emitter can be set by dragging the rotation of the object in the scene editor, or by specifying the angle in the Properties inspector. 

I am not sure if this should be added to the object properties or not.  I am leaning towards "no", since all other object types use the same method to set "angle". 

However.... while someone is working on particles... perhaps look at adding these other actions?
https://github.com/4ian/GDevelop/issues/2225
